core/docker/compose.sh

```diff
@@ -0,0 +1,11 @@
+composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+
+if [ "$1" == "up" ]; then
```

Comment:
@rcrichton, are we always going to be in dev mode?

Reply:
I'm thinking for the docker deploy we can assume dev mode for the moment. This is generally what it will be used for. I don't want to make the scripts too complex by supporting multiple modes as the might be replaced by the go app in the future.